com.microsoft.mrtk.spatialmanipulation/BoundsControl/BoundsControl.cs

```diff
@@ -278,6 +278,26 @@ public RotateAnchorType RotateAnchor
             }
         }
 
+        [EnumFlags]
+        [SerializeField]
+        [Tooltip("Specifies whether the scale handles will rotate the object around their opposing corner, or the center of its calculated bounds.")]
+        private ScaleAnchorType scaleAnchor = ScaleAnchorType.OppositeCorner;
+
+        /// <summary>
+        /// Specifies whether the scale handles will rotate the scale around the opposing corner, or the center of its calculated bounds.
+        /// </summary>
+        public ScaleAnchorType ScaleAnchor
+        {
+            get { return scaleAnchor; }
```

Comment:
Format nit, `get => scaleAnchor;`

Reply:
Agreed - I copied the style from `RotateAnchor` but will update that and `ScaleBehaviour`.